playbooks/common/openshift-cluster/evaluate_groups.yml

```diff
@@ -29,6 +29,14 @@
       msg: The nfs group must be limited to one host
     when: (groups[g_nfs_hosts] | default([])) | length > 1
 
+  - name: Evaluate oo_masters
+    add_host:
+      name: "{{ item }}"
+      groups: oo_masters
+      ansible_ssh_user: "{{ g_ssh_user | default(omit) }}"
+      ansible_sudo: "{{ g_sudo | default(omit) }}"
+    with_items: "{{ g_all_master_hosts | default([]) }}"
```

Comment:
Should we skip the definition of g_all_master_hosts in the cluster_host.yml files, and just set oo_masters to the union of g_master_hosts and g_new_master_hosts?